src/structures/ClientUser.js

```diff
@@ -82,7 +82,13 @@ class ClientUser extends User {
   edit(data, password) {
     const _data = {};
     _data.username = data.username || this.username;
-    _data.avatar = this.client.resolver.resolveBase64(data.avatar) || this.avatar;
```

Comment:
Hm... Are you sure? If I remove everything I've added, `.setAvatar('')` and `.setAvatar(null)` will end up [here](https://github.com/hydrabolt/discord.js/blob/0baa59b679d8542857926d645a02982204b3b80c/src/structures/ClientUser.js#L159).

I think, an empty string would then be resolved [here](https://github.com/hydrabolt/discord.js/blob/4292134647608206e2b3b379485a5494854c648e/src/client/ClientDataResolver.js#L204) and null would end up [here](https://github.com/hydrabolt/discord.js/blob/4292134647608206e2b3b379485a5494854c648e/src/client/ClientDataResolver.js#L214). So it would not work. Am I wrong?

Sorry, if I messed up linking the code lines. I'm new to GitHub PR change requests. ㅠㅠ